content/dstu2/immunization.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Immunization | FHIR DSTU 2 BALLOT API
+---
+
+# Immunization
+
+* TOC
+{:toc}
+
+## Search
+
+Search for Immunizations that meet supplied query parameters:
+
+    GET /Immunization?:parameters
+
+### Parameters
+
+ Name    | Required? | Type                                                           | Description
+---------|-----------|----------------------------------------------------------------|---------------------------------------------------------
+`patient`| Y |[`reference`](http://hl7.org/fhir/2015May/search.html#reference)| The patient for the vaccination record. Example: `12345`
```

Comment:
Should probably actually be http://hl7.org/fhir/dstu2/search.html#reference (so that when STU 3 comes out, the link still goes to the right version)